.github/workflows/release.yaml

```diff
@@ -0,0 +1,52 @@
+name: Release
+
+on:
+  release:
+    types: published
+
+jobs:
+  build:
+    uses: ./.github/workflows/build.yaml
+
+  release:
+    runs-on: ubuntu-latest
+    needs: build
+    permissions:
+      contents: write
+
+    steps:
+      - name: Download Artifacts
+        uses: actions/download-artifact@v4
+        with:
+          pattern: '*-release'
+          path: artifacts
+
+      - name: Release Artifact Upload
+        uses: softprops/action-gh-release@v2
+        with:
+          files: artifacts/**/*
```

Comment:
Not sure how this action works, but is there any way to strip the `-release` part from the executable name? I'd like the release artifact to just be called `wsl-usb-manager.exe`.